Creation of the Readme and Setup of the Rust workshop. First version. (#9)
PoCInnovation/Workshops: software/13.Rust/README.md

```shell
0, 1, 1, 2, 3, 5, 8, 13, 21, 34, 55, ...
```

##### Resources
 - [Fibonacci number](https://www.wikiwand.com/en/Fibonacci_number)
 - [Match](https://doc.rust-lang.org/rust-by-example/flow_control/match.html)

#

### Step 1.4 - A simple and useful `Calculator` !

> :exclamation: We strongly advise you to use the resources given for this exercise.

A little bit harder !

You will have to create a small calculator that will manage :
```
- addition
- subtraction
- multiplication
- division
```

The numbers to manipulate must be recovered as you go along and must be entered by the user (Be careful with your error management).

After your calculation you just have to display the result in your terminal.

##### Resources
 - [Module std::io](https://doc.rust-lang.org/std/io/index.html)
 - [Examples io](https://doc.rust-lang.org/std/io/struct.Stdin.html#method.read_line)
 - [Option & unwrap](https://doc.rust-lang.org/rust-by-example/error/option_unwrap.html)
 - [Match](https://doc.rust-lang.org/rust-by-example/flow_control/match.html)

#

### Step 1.5 - Pig Latin

> :exclamation: We strongly advise you to use the resources given for this exercise.

The serious stuff begins !

> 

Reviewer: In which file we must code ?

> The idea is inspired by `Pig Latin` but with a small variation nevertheless it can help you in learning a little more about the [original version](https://www.wikiwand.com/en/Pig_Latin).

Reviewer: ```suggestion
The idea is inspired by `Pig Latin` but with a small variation nevertheless it can help you  learning a little more about the [original version](https://www.wikiwand.com/en/Pig_Latin).
```

I don't understand the end ?

> From a word you must follow the following rules:

Reviewer: From a word ?

> From a word you must follow the following rules:

Reviewer: What my function must takes as arguments ?

- If it starts with a vowel then it doesn't matter, you just have to add the suffix `-hay` at the end of your word.

> - If it starts with a consonant then you must take the first letter of the word and move it to the first position of the suffix instead of the `h`.

Reviewer: I don't understand ? Just say
```suggestion
- If it starts with a consonant then you must take the first letter of the word and and the following suffix to your word `-<your letter>ay`.
```

> :bulb: We put you some examples to better visualize:

```Rust
"PoC"        -->    "oC-Pay"
"Epitech"    -->    "Epitech-hay"
"Bumblebee"  -->    "umblebee-Bay"
"Although"   -->    "Although-hay"
```

> Now that you manage words, why not tackle sentences? Apply the previous principle but on a whole sentence!

Reviewer: We must add a second function ?

> :bulb: Here are some examples too:

```Rust
"Live as if you were to die tomorrow. Learn as if you were to live forever."
-->
"ive-Lay as-hay if-hay you-hay ere-way o-tay ie-day omorrow-tay. earn-Lay as-hay if-hay you-hay ere-way o-tay ive-lay orever-fay."
```
```Rust
"A problem without a solution is a poorly stated problem"
-->
"A-hay roblem-pay ithout-way a-hay olution-say is-hay a-hay oorly-pay tated-say roblem-pay."
```

##### Resources
 - [Chars](https://doc.rust-lang.org/std/str/struct.Chars.html)
 - [Iterator](https://doc.rust-lang.org/std/iter/trait.Iterator.html)

> #

Reviewer: Why # ?

## Bonus
 - ...
 - ...
 - ...

> #

Reviewer: Why # ?

## Further reading
 - ...
 - ...
 - ...

#

## Authors
- [Yoel EDERY](https://www.linkedin.com/in/yoel-edery-957117210/)
- [Nicolas HEUDE](https://www.linkedin.com/in/nicolas-heude-525567197)

> #

Reviewer: Why # ?

## Organization

> - [📒 PoC's Linkedin](https://www.linkedin.com/company/pocinnovation/mycompany/)
> - [📷 PoC's Instagram](https://www.instagram.com/pocinnovation/)
> - [🖱️ PoC's Website](https://www.poc-innovation.fr/)
> - [:computer: PoC's Github](https://github.com/orgs/PoCInnovation/)
> - [🌐 PoC's Discord](https://discord.gg/G4Aygn6p)

Reviewer: ```suggestion
- [📒 Linkedin](https://www.linkedin.com/company/pocinnovation/mycompany/)
- [📷 Instagram](https://www.instagram.com/pocinnovation/)
- [🖱️ Website](https://www.poc-innovation.fr/)
- [🌐 Discord](https://discord.gg/G4Aygn6p)
```

They are already on the Github...
